blockchain/blockchain.go

```diff
@@ -1351,10 +1351,10 @@ func (bc *BlockChain) writeStateTrie(block *types.Block, state *state.StateDB) e
 		}
 
 		if isCommitTrieRequired(bc, block.NumberU64()) {
-			logger.Trace("Commit the state trie into the disk", "blocknum", block.NumberU64())
 			if err := trieDB.Commit(block.Header().Root, true, block.NumberU64()); err != nil {
 				return err
 			}
+			logger.Trace("Commit the state trie into the disk", "blocknum", block.NumberU64())
```

Comment:
How about using `Committed the ~`?

Reply:
Good suggestion. Before making a change, let's hold off for a bit and gather input from others to ensure consensus.

Reply:
I agree with the above comment as the timing of the log output has changed to after the commit is done.

Reply:
In addition, it's trace log, so it won't affect monitoring service. Could you proceed this comment?

Reply:
Thank you all. Updated.